### PR TITLE
Now you just need to make an empty +secret\__secret.dme

### PR DESCRIPTION
**Pull request by spookydonut**
## About the PR
Should help people get started

## Why's this needed?
Also is backwards compatible with people doing `#define SECRETS_ENABLED 0`

### CODE REVIEW

File: code/__secret.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+#ifndef SECRETS_ENABLED
+
+//Enables placeholder objects
+#include "../code/_placeholder.dm"
```
Comment (ZeWaka) [inline code/__secret.dm:5]: I mean this does work, but 😿

Comment (spookydonut) [reply code/__secret.dm:5]: this is from the build instructions

Comment (spookydonut) [inline code/__secret.dm:5]:
```suggestion
#include "_placeholder.dm"
```

File: code/__secret.dm [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+//Future Expansion
+#include "../code/_publicVersion.dm"
```
Comment (spookydonut) [inline code/__secret.dm:8]:
```suggestion
#include "_publicVersion.dm"
```

**Pull request merged**